src/legacy/core_plugins/vis_default_editor/public/components/controls/time_interval.tsx

```diff
@@ -56,15 +81,46 @@ function TimeIntervalParamEditor({
   let selectedOptions: ComboBoxOption[] = [];
   let definedOption: ComboBoxOption | undefined;
   let isValid = false;
+  let invalidEsMessage = '';
+  const errors = [];
```

Comment:
If you are using only one error message, you could get rid of `errors` array and use only `invalidEsMessage` variable for an error.